tests/index.js

```diff
@@ -170,35 +170,57 @@ test('sagaTestEngine throws under bad conditions', t => {
     () => sagaTestEngine(f, badMapping),
     'Env Mapping is missing a value for "key"')
 
+  // Bad mapping for saga that yields obj with anonymous function
+  function namedFunction() {}
+  const f2 = function*() {
+    yield { func: namedFunction }
+  }
+  const badMapping2 = [['no', 'functions']]
+  t.throws(
+    () => sagaTestEngine(f2, badMapping2),
+    `Env Mapping is missing a value for ${JSON.stringify({ "func": "[Function: namedFunction]" }, null, 2)}`
+  )
+
+  // Bad mapping for saga that yields object with anonymous function
+  var anonymousFunction = function() { return 'something'; }
+  const f3 = function*() {
+    yield { func: anonymousFunction }
+  }
+  const badMapping3 = [['also no', 'functions']]
+  t.throws(
+    () => sagaTestEngine(f3, badMapping3),
+    `Env Mapping is missing a value for ${JSON.stringify({ func: `[Function]: ${anonymousFunction.toString()}` }, null, 2)}`
+  )
```

Comment:
Hmmmmm interesting. I noticed that code run through Babel does the same thing. I might have to just remove the test unless I can think of something creative. Maybe conditionally skip the test if a function that is expected to be anonymous gets a name assigned to it.